RELEASES.md

```diff
@@ -0,0 +1,77 @@
+# Grafana Releases
+
+## Release schedule
+
+Release cadence of first beta-release being cut is 6 weeks.
+
+## Branch management and versioning strategy
+
+We use [Semantic Versioning](https://semver.org/).
+
+We maintain a separate branch for each minor release, named `v<major>.<minor>.x`, e.g. v6.0.x, v6.5.x. The release branch is created from the master branch just before  releasing the first beta-release of a minor version, e.g. v6.5.0-beta1.
+
+New features, bug fixes and changes are squashed and merged into the master branch. Bug fixes needed to be included in a patch release are being cherry-picked from the master branch. The master branch is the development branch.
+
+## Release management roles
+
+### Release coordinator
+
+Responsible for coordinating entire release series of a minor release, meaning all beta- and patch releases of a minor release.
+
+### Release responible
+
+Assigned by release coordinator or voluntarily requested. Responsible for doing the actual work of an individual release.
+
+## Create a new milestone
```

Comment:
It looks like you are starting a description of a process.

Create a new milestone
Pre-release tasks
Check milestone status
and so on

I suggest moving the level 2 headings down one level and collecting them under something that describes the "bucket" they are in, like "Release process."

You could also rewrite it as an ordered list, which would save you from having to add content to the empty heading sectins.